acceptance-tests/src/main/kotlin/org/ostelco/at/common/Endpoint.kt

```diff
@@ -4,4 +4,6 @@ package org.ostelco.at.common
 // and will be http://localhost:9090 when running in IDE connecting to prime in docker-compose
 val url: String = "http://${System.getenv("PRIME_SOCKET") ?: "localhost:9090"}"
 
-val ocsSocket = System.getenv("OCS_SOCKET") ?: "localhost:8082"
\ No newline at end of file
+val ocsSocket = System.getenv("OCS_SOCKET") ?: "localhost:8082"
+
+val pubSubEmulatorHost = System.getenv("PUBSUB_HOST") ?: "localhost:8085"
```

Comment:
Just like 2 other sockets configured above, use ENV variable passed in AT in docker-compose.
Or use localhost if running directly from IDE.